.github/workflows/kakarot_rpc.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest-32-cores
+    runs-on: ubuntu-latest-16-cores
```

Comment:
We can just use 16 cores finally with cross-compilation as it's fast enough https://github.com/alamatxyz/kakarot-rpc/actions/runs/5869304788/job/15914007439